CMakeLists.txt

```diff
@@ -1,5 +1,5 @@
 cmake_minimum_required(VERSION 3.14 FATAL_ERROR)
-PROJECT( DD4hep LANGUAGES NONE)
+PROJECT( DD4hep LANGUAGES CXX)
```

Comment:
The NONE is needed when we just want to build documentation?

Reply:
That is correct. It is later overwritten in https://github.com/AIDASoft/DD4hep/blob/f9ef7188119b272a5da02695a28bac7efecaa966/CMakeLists.txt#L120. Now it is back to NONE.